test/runnable/test11.d

```diff
@@ -1275,9 +1275,6 @@ void test8809()
         char test3Bx() { return (new class Object { char bar() { return B.foo(); } }).bar(); }
         char test3Cx() { return (new class Object { char bar() { return C.foo(); } }).bar(); }
         char test3Dx() { return (new class Object { char bar() { return   foo(); } }).bar(); }
-        char test3By() { return (new class Object { char bar() { return this.outer.B.foo(); } }).bar(); }
-        char test3Cy() { return (new class Object { char bar() { return this.outer.C.foo(); } }).bar(); }
-        char test3Dy() { return (new class Object { char bar() { return this.outer.  foo(); } }).bar(); }
```

Comment:
In old tests, the `this.outer` part was just redundant. For example, `return B.foo();` and `return this.outer.B.foo();` were completely same code.

After the bugfix, `this.outer` of those anonymous classes will have the type `void*`, the `foo` functions will be inaccessible via `void*`.